Makefile

```diff
@@ -1,5 +1,5 @@
-export TRANSIFEX_RESOURCE = frontend-app-authn
-transifex_langs = "ar,fr,es_419,zh_CN,it_IT,pt_PT,de_DE,uk,ru,hi"
+export TRANSIFEX_RESOURCE=frontend-app-learning
```

Comment:
Why does it say `frontend-app-learning`?

Reply:
Hello, I apologize, I missed the name, it has been corrected.